src/Subscription/StaticInMemorySubscriptionStoreFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\LaravelEventSourcing\Subscription;
+
+use Patchlevel\EventSourcing\Subscription\Store\InMemorySubscriptionStore;
+
+final class StaticInMemorySubscriptionStoreFactory
```

Comment:
I'm not sore if we even need this in laravel or if this is always we case due to the singleton pattern 🤔

Reply:
> When running tests, Laravel boots the application for each individual test method. 

https://laravel.com/docs/12.x/testing#configuration-caching

Seems like it should be fine to add this here too.